semcore/pagination/src/Pagination.jsx

```diff
@@ -95,7 +97,12 @@ class PaginationRoot extends Component {
 
   handlePageValueBlur = () => {
     const { currentPage } = this.asProps;
-    this.setState({ dirtyCurrentPage: currentPage });
+
+    setTimeout(() => {
+      if (this.pageInputAddonRef.current !== document.activeElement) {
+        this.setState({ dirtyCurrentPage: currentPage });
+      }
+    });
```

Comment:
Why there is no exact timeout time? I think that 400ms is too much for that wait

Reply:
my bad, added 0